### PR TITLE
Add vote results widget

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
Add vote results widget for Motions & Disputes action page

* Also extracted user avatar display splitter and counter out of ColonyMember

FireShot Capture 174 - Colony - localhost

Resolves DEV-239

### CODE REVIEW

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css
Comment (alicjakujawa) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css:81]: var(--radius-large)

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (alicjakujawa) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:1]: Hmm i dont think this component should sits under ActionsPage folder

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (alicjakujawa) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:22]: So why we need message descriptor for it at all?

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (alicjakujawa) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:26]: I honestly think its not needed as well

**Review by alicjakujawa — CHANGES_REQUESTED** [review]
Apart of small comments i see 2 ui issues:
* progress bar is too low (comparing to designs)
* Space between progress bar and avatar is too small

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (ArmandoGraterol) [reply src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:1]: Why?

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (ArmandoGraterol) [reply src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:22]: That's true

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (ArmandoGraterol) [reply src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:26]: Also true

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (alicjakujawa) [reply src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:1]: Not important anymore, thought we will have separate page for Motions

File: src/img/icons/circle-thumbs-down.svg
Comment (rdig) [inline src/img/icons/circle-thumbs-down.svg:5]: These are such bad **single-use** icons ;(

This is not your fault, it's because of the designs, and that they need to have basically three colors in them:
- green
- green w/ opacity
- grey

This breaks SVG so bad, as it prevents us from setting colors programmatically. Plus they are not really optimized. I hate them.

I've optimized the file a bit, so please replace the file with this one:
```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg viewBox="0 0 30 30"><path d="M 15,30 C 23.30039,30 30,23.30039 30,15 30,6.69961 23.30039,0 15,0 6.69961,0 0,6.69961 0,15 0,23.30039 6.69961,30 15,30 Z m 0,-1.111111 C 7.2959456,28.888889 1.1111111,22.704054 1.1111111,15 1.1111111,7.2959456 7.2959456,1.1111111 15,1.1111111 c 7.704054,0 13.888889,6.1848345 13.888889,13.8888889 0,7.704054 -6.184835,13.888889 -13.888889,13.888889 z" style="color:#000000;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:medium;line-height:normal;font-family:sans-serif;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration:none;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000000;letter-spacing:normal;word-spacing:normal;text-transform:none;writing-mode:lr-tb;direction:ltr;text-orientation:mixed;dominant-baseline:auto;baseline-shift:baseline;text-anchor:start;white-space:normal;shape-padding:0;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:0.6;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;solid-color:#000000;solid-opacity:1;vector-effect:none;fill:#68d2d2;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:1.11111116;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" /><path d="M 16.480667,12.225 V 9.5249889 C 16.480667,8.9879222 16.263889,8.4728667 15.878,8.0931111 15.492111,7.7133444 14.968778,7.5 14.423,7.5 l -2.743556,6.075 v 7.424889 h 7.736778 c 0.330889,0.0037 0.651889,-0.110445 0.903889,-0.321222 0.252111,-0.210889 0.418333,-0.504334 0.467889,-0.826223 l 0.946556,-6.075 c 0.02978,-0.193444 0.01656,-0.391 -0.03889,-0.579 C 21.640222,13.010556 21.543889,12.836667 21.413444,12.689 21.283,12.541333 21.121444,12.423333 20.94,12.343222 20.758556,12.263111 20.561556,12.222778 20.362778,12.225 Z m -4.801223,8.774889 H 9.6218222 c -0.3638222,0 -1.0287666,0 -1.3717333,0 2.22e-5,-0.337445 -3.33e-5,-0.991889 -3.33e-5,-1.35 V 14.925 c 0,-0.358111 8.88e-5,-0.675 8.88e-5,-1.35 0.6858778,0 1.0078667,0 1.3716778,0 h 2.0576218" style="fill:#68d2d2;stroke-width:1.11111116" /><path d="M 14.423828,6.9453125 A 0.55561114,0.55561114 0 0 0 13.916016,7.2714844 L 11.320312,13.019531 H 9.6210938 8.25 a 0.55561114,0.55561114 0 0 0 -0.5546875,0.554688 c 0,0.674951 0,0.993359 0,1.351562 v 4.72461 c 0,0.358168 2.22e-5,1.012272 0,1.349609 A 0.55561114,0.55561114 0 0 0 8.25,21.554688 h 1.3710938 1.9570312 a 0.55561114,0.55561114 0 0 0 0.199219,0 h 7.632812 c 0.462575,0.0051 0.911225,-0.152793 1.265625,-0.449219 0.354473,-0.296514 0.591848,-0.711622 0.66211,-1.167969 l 0.945312,-6.074219 a 0.55561114,0.55561114 0 0 0 0,-0.002 c 0.0422,-0.274119 0.02384,-0.554034 -0.05469,-0.820312 -0.07855,-0.266185 -0.214572,-0.512563 -0.398438,-0.720704 -0.183904,-0.208185 -0.411637,-0.372063 -0.666016,-0.484374 -0.254375,-0.112313 -0.528707,-0.169092 -0.80664,-0.166016 H 17.037109 V 9.5253906 c 0,-0.6865605 -0.278786,-1.3451805 -0.769531,-1.828125 -0.490727,-0.4829413 -1.153194,-0.7519531 -1.84375,-0.7519531 z m 0.302734,1.2324219 c 0.274885,0.061708 0.560575,0.1125942 0.761719,0.3105468 0.281034,0.2765668 0.4375,0.6495365 0.4375,1.0371094 v 2.6992184 a 0.55561114,0.55561114 0 0 0 0.554688,0.556641 h 3.882812 a 0.55561114,0.55561114 0 0 0 0.0059,0 c 0.119623,-0.0013 0.23719,0.0224 0.345703,0.07031 0.108509,0.04791 0.206219,0.11793 0.283203,0.205079 0.07702,0.08719 0.131722,0.189236 0.164062,0.298828 0.03236,0.109722 0.0408,0.225121 0.02344,0.33789 l -0.947266,6.074219 c -0.02886,0.187431 -0.125642,0.359111 -0.27539,0.484375 -0.1496,0.125127 -0.341812,0.195567 -0.541016,0.193359 a 0.55561114,0.55561114 0 0 0 -0.0059,-0.002 H 12.234375 V 13.695312 Z M 8.8066406,14.130859 h 0.8144532 1.5019532 v 6.3125 H 9.6210938 8.8046875 c -1.13e-5,-0.290616 0,-0.582592 0,-0.792968 v -4.72461 c 0,-0.210569 0.00193,-0.531558 0.00195,-0.794922 z" style="color:#000000;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:medium;line-height:normal;font-family:sans-serif;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration:none;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000000;letter-spacing:normal;word-spacing:normal;text-transform:none;writing-mode:lr-tb;direction:ltr;text-orientation:mixed;dominant-baseline:auto;baseline-shift:baseline;text-anchor:start;white-space:normal;shape-padding:0;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:1;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;solid-color:#000000;solid-opacity:1;vector-effect:none;fill:#f9fafa;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:1.11111116;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" /></svg>
```

Note that the above changes the size to actually be `30x30` so you might need to adjust your styles

File: src/img/icons/circle-thumbs-up.svg
Comment (rdig) [inline src/img/icons/circle-thumbs-up.svg:5]: Same as above, the icon is poorly optimized. I've re-made it, so please use this file instead:
```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg viewBox="0 0 30 30"><path d="M 15,0 C 6.6996097,0 0,6.6996092 0,15 0,23.30039 6.6996097,30 15,30 23.300391,30 30,23.30039 30,15 30,6.6996092 23.300391,0 15,0 Z m 0,1.1113281 C 22.704055,1.1113281 28.888672,7.2959456 28.888672,15 28.888672,22.704054 22.704055,28.888672 15,28.888672 7.2959461,28.888672 1.1113281,22.704054 1.1113281,15 1.1113281,7.2959456 7.2959461,1.1113281 15,1.1113281 Z" style="color:#000000;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:medium;line-height:normal;font-family:sans-serif;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration:none;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000000;letter-spacing:normal;word-spacing:normal;text-transform:none;writing-mode:lr-tb;direction:ltr;text-orientation:mixed;dominant-baseline:auto;baseline-shift:baseline;text-anchor:start;white-space:normal;shape-padding:0;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:0.6;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;solid-color:#000000;solid-opacity:1;vector-effect:none;fill:#fe5e7c;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:1.11111116;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" /><path d="m 13.519333,17.775 v 2.7 c 0,0.537111 0.216778,1.052111 0.602667,1.431889 C 14.507889,22.286667 15.031222,22.5 15.577,22.5 l 2.743556,-6.075 V 9.0000884 H 10.583767 C 10.252944,8.9964106 9.9319111,9.1105106 9.6798333,9.3213773 9.4277556,9.5322439 9.2616,9.8256662 9.212,10.147577 l -0.9465222,6.074979 c -0.029844,0.193444 -0.016578,0.391 0.038867,0.579 0.055433,0.187888 0.1517334,0.361777 0.2822223,0.509444 0.1304889,0.147667 0.2920333,0.265667 0.4734555,0.345778 0.1814222,0.08011 0.3783778,0.120444 0.5772222,0.118222 z m 4.801223,-8.7749116 h 2.057666 c 0.363778,0 1.028778,0 1.371667,0 0,0.3375 1.11e-4,0.9919444 1.11e-4,1.3499886 V 15.075 c 0,0.358111 -1.11e-4,0.675 -1.11e-4,1.35 -0.685889,0 -1.007889,0 -1.371667,0 h -2.057666" style="fill:#fe5e7c;stroke-width:1.11111116" /><path d="m 18.3125,8.4375 a 0.55561114,0.55561114 0 0 0 -0.08984,0.00781 H 10.589844 C 10.12731,8.4401705 9.6786979,8.5980043 9.3242188,8.8945312 8.9697515,9.1910482 8.7324084,9.6062496 8.6621094,10.0625 l -0.9453125,6.074219 a 0.55561114,0.55561114 0 0 0 0,0.002 c -0.042318,0.274296 -0.023826,0.554092 0.054687,0.820312 0.078523,0.26615 0.2144544,0.5125 0.3984375,0.720704 0.1838999,0.208109 0.4116633,0.37206 0.6660156,0.484374 0.2543064,0.112295 0.5285868,0.169123 0.8066406,0.166016 h 3.3203129 v 2.144531 c 0,0.686593 0.278819,1.345185 0.769531,1.828125 0.490732,0.482961 1.153202,0.751953 1.84375,0.751954 a 0.55561114,0.55561114 0 0 0 0.507812,-0.326172 l 2.595704,-5.748047 H 20.378906 21.75 a 0.55561114,0.55561114 0 0 0 0.554688,-0.554688 c 0,-0.674939 0,-0.993336 0,-1.351562 v -4.72461 c 0,-0.3581586 -10e-7,-1.0122308 0,-1.349609 A 0.55561114,0.55561114 0 0 0 21.75,8.4453125 H 20.378906 18.421875 A 0.55561114,0.55561114 0 0 0 18.3125,8.4375 Z m -7.734375,1.1171875 a 0.55561114,0.55561114 0 0 0 0.0059,0 h 7.181641 v 6.7500005 l -2.492187,5.517578 c -0.274885,-0.06171 -0.560579,-0.112592 -0.761719,-0.310547 -0.281067,-0.276616 -0.4375,-0.64948 -0.4375,-1.03711 V 17.775391 A 0.55561114,0.55561114 0 0 0 13.519531,17.21875 H 9.6367188 a 0.55561114,0.55561114 0 0 0 -0.00586,0 c -0.119635,0.0013 -0.2371651,-0.02238 -0.3457032,-0.07031 C 9.1766641,17.10053 9.079031,17.030584 9.0019531,16.943359 8.9249584,16.856229 8.8702342,16.754158 8.8378906,16.644531 8.8055148,16.534752 8.7970825,16.419233 8.8144531,16.306641 l 0.9472657,-6.074219 c 0.028901,-0.187571 0.1257024,-0.3591589 0.2753902,-0.4843751 0.149677,-0.1252065 0.341904,-0.195573 0.541016,-0.1933594 z m 8.298828,0 h 1.501953 0.814453 c 3.2e-5,0.2912267 0.002,0.5841005 0.002,0.7949215 v 4.72461 c 0,0.210556 -0.0019,0.531555 -0.002,0.794922 h -0.814453 -1.501953 z" style="color:#000000;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:medium;line-height:normal;font-family:sans-serif;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration:none;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000000;letter-spacing:normal;word-spacing:normal;text-transform:none;writing-mode:lr-tb;direction:ltr;text-orientation:mixed;dominant-baseline:auto;baseline-shift:baseline;text-anchor:start;white-space:normal;shape-padding:0;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:1;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;solid-color:#000000;solid-opacity:1;vector-effect:none;fill:#f9fafa;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:1.11111116;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" /></svg>
```

As above, note that the above changes the size to actually be `30x30` so you might need to adjust your styles

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:76]: Why are passing the `colonyAddress` here, it's really not needed in this context

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:35]: I think this should actually be `minAvatars`, so that we can configure the minum number of avatars to display

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:22]: This should be a message descriptor! 

We're not really using plain string values anywhere in dapp

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css:5]: Why does this have this **HUGE** padding ? This is too much

Screenshot from 2021-03-12 12-21-37

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css:12]: The approve/dissaprove icon should be aligned with the avatar icons.

(yes, I know that the specs are mis-aligned as well, but that doesn't mean we have to implement them as broken)

Screenshot from 2021-03-12 12-23-42

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:32]: You also need a `max` value prop otherwise you can't properly display the bar.

Screenshot from 2021-03-12 12-22-25

And it will over-run the container

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:65]: Why is this not using the `ProgressBar` that you've built in the previous PR ?

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:60]: The percent value should be calculated from the `value` and the `max` props.

We can't expect to be passed the percent value directly. How will that even work? Percent out of what ?

File: src/modules/dashboard/hooks/useAvatarDisplayCounter.ts
Comment (rdig) [inline src/modules/dashboard/hooks/useAvatarDisplayCounter.ts:1]: I think we should move all hooks in the `dashboard` module inside `utils/hooks`.

They really have no place being here.

But other then that, very nice of you for building this hook.

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css:1]: The wrapper should have a maxium limit to which it can expand, otherwise it can't be used in any places that we need it to:
- action page feed
- details sidebar

This is how it looks when using it in the feed, with padding:

Screenshot from 2021-03-12 12-32-38

Notice how it:
- breaks the container
- doesn't have right padding applied (because it doesn't have a limit)

**Review by rdig — CHANGES_REQUESTED** [review]
This is half-implemented! :(

This needs a lot more work before we can use it. 

Please be more thorough in your implementations and testing, as this is  quite a **low quality PR** and work like this is not acceptable.

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (ArmandoGraterol) [reply src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:35]: If it was `minAvatars` instead of `maxAvatars`, then that means that if I do something like `minAvatars={3}` I would expect that the widget shows **at least** 3 or more avatars? So if we have 15 voters, it would show the 15 avatars?

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css
Comment (ArmandoGraterol) [reply src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css:5]: Yeah, sorry about that. This is something I forgot to remove, from when I was testing things

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css
Comment (ArmandoGraterol) [reply src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css:12]: Checked this after changing the approve/disapprove icons to the ones you provided and they seem to be aligned. I'm gonna mention anyway that in that screenshot the upper line it's not taking into account the `1px white border` that the avatars have

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (rdig) [reply src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:37]: Yes, you are right. You can ignore this.

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css
Comment (rdig) [reply src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css:11]: Ok. I'll have a look again, taking the white border into account

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx:69]: If the `VotingResultWidget` theme is `disapprove` the progress bar should have the `danger` theme.

This is the spec:

Screenshot from 2021-03-15 22-29-45

This is how it looks currently _(only the icon changes)_

Screenshot from 2021-03-15 22-31-35

File: src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css:11]: I saw the spec, and I understand why you aligned the icon higher than the progress bar:

Screenshot from 2021-03-15 22-31-45

But I still think this will look better if the icon (and only the icon -- not the avatars) are aligned on the same line as the progress bar:

Screenshot from 2021-03-15 22-39-14

I achieved the above by applying `align-self` to the icon:
```js
.voteInfoContainer i {
  ...
  align-self: baseline;
}
```

**Review by rdig — APPROVED** [review]
Much better this time around. Very nice work! I'm now much happier with the quality of this work :100: 

There are two more things you need to change -- the progress bar theme and the icon alignments _(this last one just because I think it looks better like this, rather than how the spec is)_

But I'm also approving this since after you make the changes, you can merge.

**Pull request merged**
